dev/io.openliberty.microprofile.config.2.0.internal_fat_tck/publish/tckRunner/tck/pom.xml

```diff
@@ -43,7 +43,15 @@
                 <scope>import</scope>
             </dependency>
         </dependencies>
-    </dependencyManagement>
+    </dependencyManagement> 
+    
+	<repositories>
+		<repository>
+			<name>org.eclipse.microprofile.config</name>
+			<id>microprofile-config-tck</id>
```

Comment:
I think you should change the name and ID of the repo ... call it "IBM DHE repository" with an ID of just "DHE"